modules/nvidia-gpu-aws-adding-a-gpu-node.adoc

```diff
@@ -0,0 +1,203 @@
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                        STATUS   ROLES                  AGE     VERSION
+ip-10-0-52-50.us-east-2.compute.internal    Ready    worker                 3d17h   v1.25.4+86bd4ff
+ip-10-0-58-24.us-east-2.compute.internal    Ready    control-plane,master   3d17h   v1.25.4+86bd4ff
+ip-10-0-68-148.us-east-2.compute.internal   Ready    worker                 3d17h   v1.25.4+86bd4ff
+ip-10-0-68-68.us-east-2.compute.internal    Ready    control-plane,master   3d17h   v1.25.4+86bd4ff
+ip-10-0-72-170.us-east-2.compute.internal   Ready    control-plane,master   3d17h   v1.25.4+86bd4ff
+ip-10-0-74-50.us-east-2.compute.internal    Ready    worker                 3d17h   v1.25.4+86bd4ff
+----
+
+. View the machines and machine sets that exist in the `openshift-machine-api` namespace by running the following command.
+Each compute machine set is associated with a different availability zone within the AWS region.
+The installer automatically load balances compute machines across availability zones.
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                        DESIRED   CURRENT   READY   AVAILABLE   AGE
+preserve-dsoc12r4-ktjfc-worker-us-east-2a   1         1         1       1           3d11h
+preserve-dsoc12r4-ktjfc-worker-us-east-2b   2         2         2       2           3d11h
+----
+
+. At this time, there is only one compute machine per machine set, though a compute machine set
+could be scaled to add a node in a particular region and zone.
++
+[source,terminal]
+----
+$ oc get machines -n openshift-machine-api | grep worker
+----
++
+.Example output
++
+[source,terminal]
+----
+preserve-dsoc12r4-ktjfc-worker-us-east-2a-dts8r      Running   m5.xlarge   us-east-2   us-east-2a   3d11h
+preserve-dsoc12r4-ktjfc-worker-us-east-2b-dkv7w      Running   m5.xlarge   us-east-2   us-east-2b   3d11h
+preserve-dsoc12r4-ktjfc-worker-us-east-2b-k58cw      Running   m5.xlarge   us-east-2   us-east-2b   3d11h
+----
+
+. Make a copy of one of the existing compute `MachineSet` definitions and output the result to a JSON file by running the following command.
+This will be the basis for the GPU-enabled worker machine set definition.
++
+[source,terminal]
+----
+$ oc get machineset preserve-dsoc12r4-ktjfc-worker-us-east-2a -n openshift-machine-api -o json > <output_file.json>
+----
+
+. Edit the JSON file and make the following changes to the new `MachineSet` definition:
++
+* Replace `worker` with `gpu`. This will be the name of the new machine set.
+* Change the instance type of the new `MachineSet` definition to `g4dn`, which includes an NVIDIA Tesla T4 GPU.
+To learn more about AWS `g4dn` instance types, see https://aws.amazon.com/ec2/instance-types/#Accelerated_Computing[Accelerated Computing].
```

Comment:
the docs.openshift.com transform is happy with this, but I don't trust it not to trip the portal transform :sweat_smile: 

```suggestion
To learn more about AWS `g4dn` instance types, see link:https://aws.amazon.com/ec2/instance-types/#Accelerated_Computing[Accelerated Computing].
```